src/Input.js

```diff
@@ -5,7 +5,7 @@ import deprecationWarning from './utils/deprecationWarning';
 
 class Input extends InputBase {
   render() {
-    if (this.props.type === 'static') { // eslint-disable-line react/prop-types
+    if (this.props.type === 'static') {
```

Comment:
It is because of parallel branches :sweat: 

--
I don't like what we have with this `two branches` thing.
Too much additional burden and very doubtful profit so far.